app/Handlers/TitleRewritingHandler.php

```diff
@@ -0,0 +1,272 @@
+<?php
+
```

Comment:
suggestion: 
when you add a new class, it is wise to add this at the beginning of the file so that php does not transform the types in an uncontrollable way.

```suggestion

declare(strict_types=1);

```

Reply:
Done